pkg/files/file.go

```diff
@@ -4,11 +4,14 @@
 package files
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 	"path/filepath"
 	"sort"
 	"strings"
+
+	"github.com/k14s/ytt/pkg/literal"
```

Comment:
feels a bit weird for the lowest layer to pick up annotations from the highest layers.